tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -252,19 +253,23 @@ Check data catalogue page contains archive and superseding publication subjects
 
     user wait for option to be available and select it    css:select[id="filters-form-theme"]    %{TEST_THEME_NAME}
 
-    user checks select contains option    id:filters-form-publication    ${PUBLICATION_NAME_ARCHIVE}
-    user checks select contains option    id:filters-form-publication    ${PUBLICATION_NAME_SUPERSEDE}
+    user checks select contains option    css:select[id="filters-form-publication"]     ${PUBLICATION_NAME_ARCHIVE}
```

Comment:
To make it consistent across the PR, I have updated the locator strategy with css selectors.